### PR TITLE
Update VisibleBoundsPadding.md

### PR DESCRIPTION
**Pull request by MaximeDionNventiveCom**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

## What is the current behavior?

No example images

## What is the new behavior?

Now documentation features example Images

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] ~~Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)~~
- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] ~~[Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)~~
- [ ] ~~[Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.~~
- [x] Contains **NO** breaking changes
- [ ] ~~Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)~~
- [ ] ~~Associated with an issue (GitHub or internal)~~

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/features/VisibleBoundsPadding.md [hunk 1]
```diff
@@ -42,6 +42,13 @@ Usage is as follows:
...
 
-**Note: 
+#### Note: 
 - This behavior applies the greater of the existing padding or the calculated padding set on the element it is attached to.
-**
+
+
+## See it in action
+
+![image](https://user-images.githubusercontent.com/36631443/68240880-ac96de80-ffdb-11e9-9714-bd635f637443.png)
```
Comment (agneszitte-nventive) [inline doc/articles/features/VisibleBoundsPadding.md]: This one need to be updated please

File: doc/articles/features/VisibleBoundsPadding.md [hunk 2]
```diff
@@ -42,6 +42,13 @@ Usage is as follows:
...
 - This behavior applies the greater of the existing padding or the calculated padding set on the element it is attached to.
-**
+
+
+## See it in action
+
+![image](https://user-images.githubusercontent.com/36631443/68244085-0d291a00-ffe2-11e9-89a9-dcd66141e4f8.png)
+
+![image](https://user-images.githubusercontent.com/36631443/68239478-06e27000-ffd9-11e9-956a-c4d954341a5f.png)
+![image](https://user-images.githubusercontent.com/36631443/68239503-1366c880-ffd9-11e9-9d67-f95050b2e1a7.png)
```
Comment (carldebilly) [inline doc/articles/features/VisibleBoundsPadding.md]: You should add _before_ / _after_

File: doc/articles/features/VisibleBoundsPadding.md [hunk 3]
```diff
@@ -42,6 +42,16 @@ Usage is as follows:
...
 
-**Note: 
+#### Note: 
 - This behavior applies the greater of the existing padding or the calculated padding set on the element it is attached to.
-**
+
+
+## See it in action
+
+![image](https://user-images.githubusercontent.com/36631443/68246487-defa0900-ffe6-11e9-8e5b-2beaea7bdc51.png)
```
Comment (jeromelaban) [inline doc/articles/features/VisibleBoundsPadding.md]: It would be best if the images were local to the repository, in the assets folder.

**Comment by carldebilly** [comment]
Next time, please follow nventive's convention for development branches... `dev/<initials>/<something>`.

**Pull request merged**